parm/config/gefs/config.fcst

```diff
@@ -14,6 +14,12 @@ case ${WAVE_CDUMP} in
   *) DO_WAVE="NO" ;; # Turn waves off
 esac
 
+# Turn off waves if not used for this CDUMP
```

Comment:
```suggestion
# Turn off waves if not used for this RUN
```

Reply:
Why do aerosols need to key off of RUN and waves off of CDUMP?  Or should they both key off of RUN?

Reply:
They should both key off `RUN`. We are trying to eliminate CDUMP except in a select few instances related to observations (see #1299).

Reply:
Changed to RUN.